docs/index.md

```diff
@@ -1,6 +1,10 @@
+<!--meta-description
+Mithril.js is a modern, small, fast client-side Javascript framework for building Single Page Applications.
+-->
+
 # Introduction
 
-- [What is Mithril?](#what-is-mithril?)
+- [What is Mithril.js?](#what-is-mithril)
```

Comment:
Does the anchor still work? See below.

Reply:
It won't. I *think* this suggestion is correct, but it might be `mithril-js` - you'd have to check how links are slugified to confirm.

```suggestion
- [What is Mithril.js?](#what-is-mithriljs)
```